cli/commands/new.js

```diff
@@ -46,6 +46,8 @@ function createPackage(packageName) {
 
   shell.exec(`echo "'Hello World' -> print" > index.clio`);
 
+  shell.exec(`echo ".clio-cache\nclio_env" > .gitignore`);
```

Comment:
shelljs provides an `echo` method that wraps the command for different operating systems. I notice that I didn't do that for the command above it either. Could you substitute `shell.exec()` with `shell.echo()` in both cases?

Reply:
Yes, I'll do it.

Reply:
Just something, `shell.echo` only prints to `stdout`. I am not able to point it to a file. But instead, I can try this: `shell.ShellString('something\n').to('output.txt')`.

Reply:
Ah, thats probably why I used exec in the first place!😄 Don't mind my comment. Merged